### ISSUE TITLE
Unable to add a new domain to a certificate (urn:acme:error:malformed)

### ISSUE THREAD

**Issue by hlidotbe** [certbot/certbot#3283]
Hi,

I'm trying to expand a certificate with a new domain but it fails with the following message:

> Error: urn:acme:error:malformed :: The request message was malformed :: Unable to update challenge :: Response does not complete challenge

The domains list:

> git.epic.net,auxilium.epic.net,billings.epic.net,discourse.epic-sys.io,git.epic-sys.io,git.epic.net,registry.epic-sys.io,reports.epic.net,support.epic.net,track.epic.net,discourse.epic.net

It seems that _discourse.epic.net_ makes the request fail, certbot and lego gives the same result.

Attached is the full log for the session. The domain resolves correctly, I'm not sure what to do.

[letsencrypt.txt](https://github.com/certbot/certbot/files/361056/letsencrypt.txt)

**Comment by pde** [comment]
@jsha @rolandshoemaker is this a boulder issue?

**Comment by hlidotbe** [comment]
Any update on this? I tried again but same result using certbot 0.8.1

**Comment by ohemorange** [comment]
Is this still an issue?

**Comment by jsha** [comment]
Nope! I believe this was an occasional misleading error we would show when there was an internal error, but I think it's since been fixed.
